msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ClientAssertion.java

```diff
@@ -13,7 +12,7 @@
 @EqualsAndHashCode
 final class ClientAssertion implements IClientAssertion {
 
-    static final String assertionType = JWTAuthentication.CLIENT_ASSERTION_TYPE;
+    static final String ASSERTION_TYPE = "urn:ietf:params:oauth:client-assertion-type:jwt-bearer";
```

Comment:
Where is this constant used?

Reply:
It was supposed to be used where your other comment said to put it (https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/941#discussion_r2059554828), but must've gotten lost in a refactor.

In the latest commit it's now used in `TokenRequestExecutor`